keyboards/dz60/keymaps/chesterarrow/keymap.c

```diff
@@ -0,0 +1,96 @@
+#include "dz60.h"
+#define MODS_CTRL_MASK  (MOD_BIT(KC_LSHIFT)|MOD_BIT(KC_RSHIFT))
+
+enum custom_keycodes {
+    CHESTER_EMAIL = SAFE_RANGE,
+	ICON_EMAIL
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	KEYMAP_DIRECTIONAL(
+		KC_ESC, KC_1, KC_2, KC_3, KC_4, KC_5, KC_6, KC_7, KC_8, KC_9, KC_0, KC_MINS, KC_EQL, KC_NO, KC_BSPC, 
+		KC_TAB, KC_Q, KC_W, KC_E, KC_R, KC_T, KC_Y, KC_U, KC_I, KC_O, KC_P, KC_LBRC, KC_RBRC, KC_BSLS, 
+		LT(1, KC_CAPS), KC_A, KC_S, KC_D, KC_F, KC_G, KC_H, KC_J, KC_K, KC_L, KC_SCLN, KC_QUOT, KC_ENT, 
+		KC_LSFT, KC_NO, KC_Z, KC_X, KC_C, KC_V, KC_B, KC_N, KC_M, KC_COMM, KC_DOT, SFT_T(KC_SLSH), KC_UP, KC_SLSH, 
+		KC_LCTL, KC_LGUI, KC_LALT, KC_SPC, KC_SPC, KC_SPC, KC_RALT, MO(1), KC_LEFT, KC_DOWN, KC_RGHT),
+
+	KEYMAP_DIRECTIONAL(
+		KC_GRV, KC_F1, KC_F2, KC_F3, KC_F4, KC_F5, KC_F6, KC_F7, KC_F8, KC_F9, KC_F10, KC_F11, KC_F12, KC_TRNS, KC_DEL, 
+		KC_TRNS, RGB_TOG, KC_UP, RGB_HUI, RGB_HUD, RGB_SAI, RGB_SAD, RGB_VAI, RGB_VAD, RGB_MOD, KC_PAUS, KC_HOME, KC_END, KC_PSCR, 
+		KC_TRNS, KC_LEFT, KC_DOWN, KC_RGHT, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_PGUP, KC_PGDN, KC_TRNS, 
+		KC_TRNS, KC_TRNS, CHESTER_EMAIL, ICON_EMAIL, BL_DEC, BL_TOGG, BL_INC, BL_STEP, KC_TRNS, KC_TRNS, KC_INS, KC_TRNS, KC_VOLU, KC_TRNS, 
+		KC_TRNS, RESET, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_MUTE, KC_VOLD, KC_RCTL),
+
+};
+
+
+const macro_t *action_get_macro(keyrecord_t *record, uint8_t id, uint8_t opt) {
+
+
+	switch (id) {
+
+	}
+	return MACRO_NONE;
+}
+
+void matrix_init_user(void) {
+}
+
+void matrix_scan_user(void) {
+}
+
+// New Style Macros
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    
+	if (record->event.pressed) 
+	{
+        switch(keycode) 
+		{
+            case CHESTER_EMAIL:
+                SEND_STRING("chestercristobal@gmail.com");
+                return false; 
+				break;
+				
+			case ICON_EMAIL:
+                SEND_STRING("iconstellationi@gmail.com");     
+				return false; 
+				break;			
+        }
+    }
+    return true;
+};
+
+void led_set_user(uint8_t usb_led) {
+
+	if (usb_led & (1 << USB_LED_NUM_LOCK)) {
+		
+	} else {
+		
+	}
+
+	if (usb_led & (1 << USB_LED_CAPS_LOCK)) {
+		DDRB |= (1 << 2); PORTB &= ~(1 << 2);
```

Comment:
Could you switch these to:
```suggestion
		writePinLow(B2);
```

These are generalized pin commands, that will work for both ARM and AVR, and are cleaner/easier to read.